nbbuild/templates/common.xml

```diff
@@ -82,6 +82,16 @@
             <available file="${nbplatform.active.dir}/platform/core/asm-8.0.1.jar"/>
         </condition>
         <property name="asm.jar" location="${platform/libs.asm.dir}/core/asm-8.0.1.jar"/>
+       <condition property="tsaurl" value="">
```

Comment:
These conditionals is not needed. Use ```<property name="tsaurl" value=""/>``` instead.
Ant property processing rule is: first set value matters.